### ISSUE TITLE
Possible to find camera's pose?

### ISSUE THREAD

**Issue by eyildiz-ugoe** [ros-perception/ar_track_alvar#73]
Is the reverse functionality also provided? Like, we can find the R and T vectors of the camera extrinsics? In other words, pose of the camera?

**Comment by Victor-ayala** [comment]
No, there's not. But, if you know the markers pose you can do the inverse transformation between tags and world to know camera pose.
